### PR TITLE
Simulation tutorial

### PR DESCRIPTION
**Pull request by gtep96**
Added tutorials and quick start guide

### CODE REVIEW

File: docs_citros_web/simulations/sim_quick_start.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+   Click the ```Run Simulation``` button located at the top-right of the page.
+
+   ![png](img/quick2.png "Final Countdown")
+
+7. Setting Up the Simulation:
+
+   Ensure the appropriate Repository and Branch are selected, choose the simulation setup, and specify the number of runs and parallel launches.
+
+   ![png](img/quick3.png "Ignition!")
```
[the remark this answers is not in the document]
Comment (mariaBer) [reply docs_citros_web/simulations/sim_quick_start.md]: I think it's something old and irreelevant

File: docs/simulations/sim_overview.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+   |Term	|Description |
+   |--|--|
+   Project	|A user’s ROS-based project that includes a prepared Docker container file.
+   Project's repository	|A user’s git repository, which must be initialized and synchronized with the local workspace prior to CITROS initialization.
+   CITROS repository |A git repositroy hosted on the CITROS cloud. Manages all data related to CITROS, and synchronized with the Project's git repository during CITROS initialization.
+   Simulation Batch |A set of one or more simulation runs, executed collectively with a designated configuration and parameter setup to efficiently analyze multiple scenarios or iterations.
+   Simulation Run |A singular execution of a simulation model, performed with a specific set of parameters and conditions to analyze and observe system behavior in a specific scenario or iteration.
+
+## Table of Contents
+1. [Main Runs Page](/docs/simulations/sim_runs_page.md)
```
Comment (mariaBer) [inline docs/simulations/sim_overview.md:16]: Change to the titles of the pages

File: docs/simulations/sim_runs_page.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+   * Project name;
+   * Simulation JSON file (corresponding to the ROS Launch file);
+   * User-defined simulation name;
+   * Batch ID;
+   * Simulation launch timestamp;
+   * Number of runs and their status.
+   * Username;
+   * Commit ID (The commit the data was created with);
+   * Simulation image tag (The image used for simulation);
+   * [Status of the simulation data (results)](#simulation-status-data);
```
Comment (mariaBer) [inline docs/simulations/sim_runs_page.md:41]: Link not working

File: docs/simulations/sim_runs_page.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+## Available Actions
+
+### Search Box
+
+   The search field is available right above the Batch Run List. You can search by project's name, batch name, batch message etc. This allows to search a batch run name by entering it in the text box.
+
+### Run Simulation Button
+
+   Pressing on the button opens a window to define and run a new simulation. For more information follow this [Step-By-Step Simulation tutorial](https://citros.io/doc/docs/simulations/sim_step_by_step).
```
Comment (mariaBer) [inline docs/simulations/sim_runs_page.md:64]: Link not working

File: docs/simulations/sim_runs_page.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+![png](img/runs4.png "Run Simulation")
```
Comment (mariaBer) [inline docs/simulations/sim_runs_page.md:66]: remove image

File: docs/simulations/sim_step_by_step.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+   2.5 Define in parallelism filed, how many simulations will run in parallel in each iteration (parallelism <= repetitions).
+
+   ![png](img/quick3.png "Ignition!")
+
+3. Click on ```Run Simulation``` button.
+
+   ![png](img/quick4.png "Lift-Off!")
+
+Congratulations! You have successfully launched your first simulation batch via CITROS Web!
+Your batch now appears at the top of the batch list, indicated by a Loading status. You may initiate another simulation or visit the [Batch Page](https://citros.io/doc/docs_citros_web/simulations/sim_batch_page) to monitor its progress.
```
Comment (mariaBer) [inline docs/simulations/sim_step_by_step.md:31]: Your batch now appears at the top of the [batch list](add link to the location), indicated by a Loading status.

**Review by mariaBer — CHANGES_REQUESTED** [review]
Please dont resolve the comments instead -reply done

File: docs/simulations/sim_overview.md (hunk 2 above)
Comment (gtep96) [reply docs/simulations/sim_overview.md:16]: done

File: docs/simulations/sim_runs_page.md (hunk 3 above)
Comment (gtep96) [reply docs/simulations/sim_runs_page.md:20]: done

File: docs/simulations/sim_runs_page.md (hunk 5 above)
Comment (gtep96) [reply docs/simulations/sim_runs_page.md:45]: done

File: docs/simulations/sim_step_by_step.md (hunk 6 above)
Comment (gtep96) [reply docs/simulations/sim_step_by_step.md:28]: done

**Comment by gtep96** [comment]
@mariaBer pushed last fixes

File: docs/simulations/sim_step_by_step.md (hunk 6 above)
Comment (mariaBer) [reply docs/simulations/sim_step_by_step.md]: meant for something else, the change is minor so I change it myself

**Review by mariaBer — APPROVED** [review]

**Comment by mariaBer** [comment]
@gtep96  There are a few fixes I'm adding (mostly links) and I add it to the main

**Pull request merged**